eclair-core/src/test/scala/fr/acinq/eclair/channel/states/e/NormalStateSpec.scala

```diff
@@ -2409,9 +2409,7 @@ class NormalStateSpec extends TestkitBaseClass with StateTestsHelperMethods {
     sender.send(alice, INPUT_DISCONNECTED)
     assert(relayerA.expectMsgType[Status.Failure].cause.asInstanceOf[AddHtlcFailed].paymentHash === htlc1.paymentHash)
     assert(relayerA.expectMsgType[Status.Failure].cause.asInstanceOf[AddHtlcFailed].paymentHash === htlc2.paymentHash)
-    val update2a = alice2bob.expectMsgType[ChannelUpdate]
-    assert(channelUpdateListener.expectMsgType[LocalChannelUpdate].channelUpdate === update2a)
-    assert(!Announcements.isEnabled(update2a.channelFlags))
```

Comment:
Looks like this was catching a superfluous `ChannelUpdate` on entering into `OFFLINE` state, it's not sent now.